### ISSUE TITLE
BUG: setup.py in 2.0.1 is broken

### ISSUE THREAD

**Issue by sjsrey** [pysal/esda#46]
README.md does not come along for the ride:
```
(esdabroke) ~/D/e/dist ❯❯❯ pip install esda-2.0.1.tar.gz
Processing ./esda-2.0.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-z8gub9r6/setup.py", line 7, in <module>
        with open('README.md', 'r', encoding='utf8') as file:
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-req-build-z8gub9r6/

```

This is in the github release for [2.0.1](https://github.com/pysal/esda/releases/tag/v2.0.1). We have not yet released 2.0.1 on pypi however. I think we should cut a new bug fix release that fixes this (2.0.2) and release it on gh and pypi. In other words since 2.0.1 won't install, we should not release it on pypi.

**Comment by sjsrey** [comment]
@weikang9009 did you try to install the tarball outside of your clone? I think the clone directory has the README.md file, while the tarball in 2.0.1 does not:

```
(esdabroke) ~/D/e/dist ❯❯❯ tar xzvf esda-2.0.1.tar.gz                                                                                                               ✘ 9 
esda-2.0.1/
esda-2.0.1/MANIFEST.in
esda-2.0.1/PKG-INFO
esda-2.0.1/README.rst
esda-2.0.1/esda/
esda-2.0.1/esda/__init__.py
esda-2.0.1/esda/gamma.py
esda-2.0.1/esda/geary.py
esda-2.0.1/esda/getisord.py
esda-2.0.1/esda/join_counts.py
esda-2.0.1/esda/mixture_smoothing.py
esda-2.0.1/esda/moran.py
esda-2.0.1/esda/smoothing.py
esda-2.0.1/esda/tabular.py
esda-2.0.1/esda/tests/
esda-2.0.1/esda/tests/__init__.py
esda-2.0.1/esda/tests/test_gamma.py
esda-2.0.1/esda/tests/test_geary.py
esda-2.0.1/esda/tests/test_getisord.py
esda-2.0.1/esda/tests/test_join_counts.py
esda-2.0.1/esda/tests/test_mixture_smoothing.py
esda-2.0.1/esda/tests/test_moran.py
esda-2.0.1/esda/tests/test_smoothing.py
esda-2.0.1/esda.egg-info/
esda-2.0.1/esda.egg-info/PKG-INFO
esda-2.0.1/esda.egg-info/SOURCES.txt
esda-2.0.1/esda.egg-info/dependency_links.txt
esda-2.0.1/esda.egg-info/requires.txt
esda-2.0.1/esda.egg-info/top_level.txt
esda-2.0.1/requirements.txt
esda-2.0.1/requirements_docs.txt
esda-2.0.1/requirements_tests.txt
esda-2.0.1/setup.cfg
esda-2.0.1/setup.py
(esdabroke) ~/D/e/dist ❯❯❯ 
```

**Comment by weikang9009** [comment]
I've also tried to install using the tarball in the [github release](https://github.com/pysal/esda/releases/tag/v2.0.1). It was also successful.

**Comment by sjsrey** [comment]
I see what is happening. Yes, the README.md is in that tarball. Where I run into problems is as follows:

1. extract that tarball
2. cd esda-2.0.1
3. python setup.py sdist

then the tarball in that `dist` directory won't have the `README.md` file.

I stumbled on this as I wanted to push a release up to pypi so I grabbed our tarball and did those three steps. Then when trying to pip install I hit the problem.

I guess the best way to do this would be download our tarball off of github and push that directly up to pypi.
